test/util/authenticate-user.js

```diff
@@ -31,14 +32,14 @@ async function oidcAuthFor(service, user) {
 
   // custom cookie jar probably not important, but we will need these cookies
   // for the final redirect
-  const cookieJar = new makeFetchCookie.toughCookie.CookieJar();
+  const cookieJar = new fetchCookie.toughCookie.CookieJar();
   res1.headers['set-cookie'].forEach(cookieString => {
     cookieJar.setCookie(cookieString, 'http://localhost:8383/v1/oidc/login');
   });
 
   const location1 = res1.headers.location;
 
-  const fetchC = makeFetchCookie(fetch, cookieJar);
+  const fetchC = fetchCookie(fetch, cookieJar);
```

Comment:
nit:

I was trying to understand the rationale for the name change.

In the end, `fetchCookie` is less surprising than the previous `makeFetchCookie`.

I think the original `makeFetchCookie` was taken from the `fetch-cookie` docs, where most of the examples use `makeFetchCookie`: https://www.npmjs.com/package/fetch-cookie?activeTab=readme

pro new name: it's a better name
pro old name: less changes to consider in this PR, less git history churn